nilearn/input_data/multi_nifti_masker.py

```diff
@@ -55,6 +55,11 @@ class MultiNiftiMasker(NiftiMasker, CacheMixin):
         their mean is put to 0 and their variance to 1 in the time dimension.
         Default=True.
 
+    high_variance_confounds : boolean, optional
+        Boolean indicating whether high-variance confounds should be computed
+        from provided image and regressed out.
```

Comment:
Maybe indicate that when True, high_variance_confounds are computed with the default parameters + link to the relevant documentation: it is important for users to know how these confounds will be computed.